src/client/delphi_epidata.js

```diff
@@ -1,693 +1,621 @@
-"use strict";
-
-function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
-
-function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } }
-
-function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); return Constructor; }
-
 // Generated by CoffeeScript 2.5.1
-(function () {
+(function() {
   /*
   A module for DELPHI's Epidata API.
-   https://github.com/cmu-delphi/delphi-epidata
-   Notes:
+
+  https://github.com/cmu-delphi/delphi-epidata
+
+  Notes:
    - If running in node.js (or using browserify), there are no external
      dependencies. Otherwise, a global jQuery object named `$` must be available.
   */
-  var Epidata, https, querystring; // Use built-in node.js modules unless jQuery is available
+  var Epidata, https, querystring;
 
+  // Use built-in node.js modules unless jQuery is available
   if ((typeof $ !== "undefined" && $ !== null ? $.getJSON : void 0) == null) {
     https = require('https');
     querystring = require('querystring');
   }
 
-  Epidata = function () {
-    var BASE_URL, _list, _listitem, _request; // Because the API is stateless, the Epidata class only contains static methods
-
+  Epidata = (function() {
+    var BASE_URL, _list, _listitem, _request;
 
-    var Epidata = /*#__PURE__*/function () {
-      function Epidata() {
-        _classCallCheck(this, Epidata);
+    // Because the API is stateless, the Epidata class only contains static methods
+    class Epidata {
```

Comment:
The Delphi primary doesn't speak ES6; you'll need to transpile to ES5. This will do it:

```
babel delphi_epidata.es6.js --presets=@babel/env > delphi_epidata.js
```

Reply:
I'm confused, can you elaborate? AFAIK the server just hands this file out like any other static web content. Was there an issue with ES6 at some point that I missed?

Reply:
The server runs uglifyjs on this file. If it's in ES6, we get an error like this during deployment:

```
  [uglifyjs '/home/automation/driver/github_deploy_repo__tmp/src/client/delphi_epidata.js' -c -m -o '/home/automation/driver/github_deploy_repo__tmp/src/client/delphi_e$
idata.min.js']
Parse error at /home/automation/driver/github_deploy_repo__tmp/src/client/delphi_epidata.js:24,10
SyntaxError: Unexpected token: name (Epidata)
```

Reply:
Ohhh that's right... /facepalm. Thanks for the reminder.

At risk of starting another side quest from my current side quest, I'd like to update deployment so that the minify action uses something more modern than uglify, like babel. But I'll def leave that alone in this PR. In the meantime, I'll just recompile this targeting ES5.

Reply:
Created https://github.com/cmu-delphi/github-deploy-repo/issues/16 to track the ES6 issue.